code/datums/ai/basic_mobs/basic_ai_behaviors/set_travel_destination.dm

```diff
@@ -0,0 +1,14 @@
+/datum/ai_behavior/set_travel_destination
+	var/location_key = BB_TRAVEL_DESTINATION
```

Comment:
Probably better to pass this in from the subtree along with `target_key`, then if you need to override which key is used you don't need to go two extension levels deep.